src/modules/settings/components/settings/global/Emotes.jsx

```diff
@@ -35,9 +38,54 @@ function SafetyWarningModal({open, onClose}) {
   );
 }
 
+const EMOTE_MODIFIERS_DESCRIPTION = {
+  'w!': formatMessage({defaultMessage: 'Will display the emote in a wide format'}),
+  'h!': formatMessage({defaultMessage: 'Will flip the emote horizontally'}),
+  'v!': formatMessage({defaultMessage: 'Will flip the emote vertically'}),
+  'z!': formatMessage({defaultMessage: 'Will display zero-width between prior emote'}),
+  'l!': formatMessage({defaultMessage: 'Will rotate the emote left'}),
+  'r!': formatMessage({defaultMessage: 'Will rotate the emote right'}),
+  'c!': formatMessage({defaultMessage: 'Will display the emote in a cursed format'}),
+  'p!': formatMessage({defaultMessage: 'Will display the emote in a party format'}),
+};
+
+function EmoteModifiersModal({open, onClose}) {
+  return (
+    <Modal backdrop="static" open={open} onClose={() => onClose()} size="xs">
+      <Modal.Header>
+        <strong>{formatMessage({defaultMessage: 'Emote Modifiers'})}</strong>
+      </Modal.Header>
+      <Modal.Body>
+        <p className={emotesStyles.modifiersModalDescription}>
+          {formatMessage({
+            defaultMessage:
+              'Emote modifiers allow you to transform emotes in realtime. To use them, simply add the modifier to the start of the emote code. For example, w! Kappa will display the emote in a wide format.',
```

Comment:
The example should probably use a global BTTV emote, not Kappa, since the modifier would not actually work on Kappa. Might I recommend `w! monkaS`?

I might also rephrase "add the modifier to the start of the emote code" to "add the modifier directly before the emote".